### ISSUE TITLE
UnicodeEncodeError when click on fulltext icon

### ISSUE THREAD

**Issue by luohongliang** [inpho/topic-explorer#208]
The fulltext function doesn't work on [website](http://inpho.xjtu.edu.cn/handianouyang/60/?doc=handian%2F%E3%80%8E%E5%8F%B2%E9%83%A8%E3%80%8F%2F%E6%AD%A3%E5%8F%B2%2F%E5%8D%97%E9%BD%90%E4%B9%A6%2F%E5%8D%B7%E4%BA%8C%E5%8D%81%E4%BA%94%E5%88%97%E4%BC%A0%E7%AC%AC%E5%85%AD%E3%80%80%E5%9E%A3%E5%B4%87%E7%A5%96%E3%80%80%E5%BC%A0%E6%95%AC%E5%84%BF.txt)
But it works fine in localhost

my version is: 1.0b169-0-gc7a7961

the error log shows:
```
[Wed Aug 16 21:23:21.440172 2017] [:error] [pid 12125:tid 140039366493952] [remote 202.200.235.51:50303] Traceback (most recent call last):
[Wed Aug 16 21:23:21.440233 2017] [:error] [pid 12125:tid 140039366493952] [remote 202.200.235.51:50303]   File "/home/inpho/anaconda2/lib/python2.7/site-packages/bottle.py", line 866, in _handle
[Wed Aug 16 21:23:21.440240 2017] [:error] [pid 12125:tid 140039366493952] [remote 202.200.235.51:50303]     return route.call(**args)
[Wed Aug 16 21:23:21.440246 2017] [:error] [pid 12125:tid 140039366493952] [remote 202.200.235.51:50303]   File "/home/inpho/anaconda2/lib/python2.7/site-packages/bottle.py", line 1744, in wrapper
[Wed Aug 16 21:23:21.440251 2017] [:error] [pid 12125:tid 140039366493952] [remote 202.200.235.51:50303]     rv = callback(*a, **ka)
[Wed Aug 16 21:23:21.440258 2017] [:error] [pid 12125:tid 140039366493952] [remote 202.200.235.51:50303]   File "/home/inpho/topicexplorer/topicexplorer/server.py", line 51, in set_header
[Wed Aug 16 21:23:21.440263 2017] [:error] [pid 12125:tid 140039366493952] [remote 202.200.235.51:50303]     return f(*args, **kwargs)
[Wed Aug 16 21:23:21.440267 2017] [:error] [pid 12125:tid 140039366493952] [remote 202.200.235.51:50303]   File "/home/inpho/topicexplorer/topicexplorer/server.py", line 487, in get_doc
[Wed Aug 16 21:23:21.440272 2017] [:error] [pid 12125:tid 140039366493952] [remote 202.200.235.51:50303]     return static_file(doc_id, root=fdirs)
[Wed Aug 16 21:23:21.440277 2017] [:error] [pid 12125:tid 140039366493952] [remote 202.200.235.51:50303]   File "/home/inpho/anaconda2/lib/python2.7/site-packages/bottle.py", line 2475, in static_file
[Wed Aug 16 21:23:21.440281 2017] [:error] [pid 12125:tid 140039366493952] [remote 202.200.235.51:50303]     if not os.path.exists(filename) or not os.path.isfile(filename):
[Wed Aug 16 21:23:21.440286 2017] [:error] [pid 12125:tid 140039366493952] [remote 202.200.235.51:50303]   File "/usr/lib/python2.7/genericpath.py", line 18, in exists
[Wed Aug 16 21:23:21.440291 2017] [:error] [pid 12125:tid 140039366493952] [remote 202.200.235.51:50303]     os.stat(path)
[Wed Aug 16 21:23:21.440295 2017] [:error] [pid 12125:tid 140039366493952] [remote 202.200.235.51:50303] UnicodeEncodeError: 'ascii' codec can't encode characters in position 28-31: ordinal not in range(128)
```

**Comment by JaimieMurdock** [comment]
The link is working for me - were you able to update or does the issue still exist?

**Comment by JaimieMurdock** [comment]
Hi @luohongliang I think we just got this resolved in 1.0b175 and up. Let me know if it is still a problem
